nibabies/workflows/bold/base.py

```diff
@@ -398,6 +466,9 @@ def init_func_preproc_wf(bold_file, has_fieldmap=False):
     )
     val_bold.inputs.in_file = listify(bold_file)
 
+    # Select validated BOLD files (orientations checked or corrected)
+    validated_bold = pe.Node(niu.Select(), run_without_submitting=True, name="validated_bold")
```

Comment:
It's a little annoying to have this along with `val_bold`, but since we pass the first value of `val_bold` to HMC workflow, we're going to need a second node either way. The alternative would be pass `bold_source` to `val_bold` (now a `Node` that gets iterated over), and then something like `JoinNode(Select(index=0), joinfield=["inlist"])` to pass to HMC. That's probably a little cleaner, but then more divergent to vanilla fMRIPrep. Your call.

Reply:
I think the node names can be a little clearer - what about:
- `val_bold` -> `validate_bolds`
- `validated_bold` -> `select_bold`